src/nidmm/nidmm.mak

```diff
@@ -8,6 +8,15 @@ MODULE_FILES_TO_COPY := $(DEFAULT_PY_FILES_TO_COPY)
 
 RST_FILES_TO_GENERATE := $(DEFAULT_RST_FILES_TO_GENERATE)
 
+# nidmm has been chosen to represent all the drivers for the purpose
```

Comment:
This is no longer true, right?

Reply:
This file can now be built by any driver.